### PR TITLE
Add site content

### PR DESCRIPTION
**Pull request by itsdeannat**
Adds all the website content, including blog content and images

### CODE REVIEW

File: content/blog/basic-commands.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+preview: /uploads/looking_dog.jpg
+---
+
+Some new dog owners think that they don't need to start teaching their dogs any commands until they reach a certain age. It's actually very important to start teaching your dog certain actions, or commands, as early as possible. Starting your training at an early age can help to build a solid foundation between you and your dog. 
+
+There are several basic commands that every dog owner should teach their dog. These commands can help you manage some common behavioral problems, like growling or trying to slip off their leash. 
+
+## Coming to you
+
+Sometimes your dog may slip off his leash if he sees a squirrel running by or escape into the backyard while you're taking out the trash. In these instances, it's best to use the come command to direct your dog towards you. If you plan on taking your dog off his leash, this is an essential command to know.
```
Comment (Begster) [inline content/blog/basic-commands.md:19]: Suggestion: should the commands be italicized?

Comment (Begster) [inline content/blog/basic-commands.md:19]: Suggestion/ question: Should we call attention to all dog commands by highlighting them by bolding or italicizing them? Such as, "...use the _come_ command to direct your do towards you." or maybe "...use the **_come_** command to direct your do towards you."

File: content/blog/basic-commands.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## Coming to you
+
+Sometimes your dog may slip off his leash if he sees a squirrel running by or escape into the backyard while you're taking out the trash. In these instances, it's best to use the come command to direct your dog towards you. If you plan on taking your dog off his leash, this is an essential command to know. 
+
+## Staying in place
+
+The stay command can be difficult for dogs to master. However, this is an essential command for your dog to know. Sometimes, you need your dog to stay in one place. For example, if you drop something sharp on the floor, you might use the stay command to make sure your dog remains still while you clean up. 
+
+If you're the proud owner of a new puppy, you might find this [blog post(https://www.rover.com/blog/train-puppy-stay/) by our friends at Rover.com helpful.
```
Comment (Begster) [inline content/blog/basic-commands.md:25]: Suggestion: I think a closing square bracket is missing for the text "blog post".

File: content/blog/tips-for-your-first-dog.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+title: 4 tips for picking your first dog
+date: 2022-03-04T23:34:20.818Z
+draft: false
+type: blog
+lastmod: 2022-03-25T18:43:34.615Z
+description: There are a few considerations to keep in mind.
+preview: /uploads/Listening.jpg
+---
+
+Congratulations on deciding to become a dog owner! Owning a dog is a wonderful, rewarding experience. Dogs make great companions, they're fun, and if you pick a large dog, they can be great body pillows.
```
Comment (Begster) [inline content/blog/tips-for-your-first-dog.md:11]: "...and if you pick a large dog, they can be great body pillows."
I love this! 😂

File: content/blog/tips-for-your-first-dog.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+Congratulations on deciding to become a dog owner! Owning a dog is a wonderful, rewarding experience. Dogs make great companions, they're fun, and if you pick a large dog, they can be great body pillows. 
+
+The first step to becoming a dog owner, of course, is to pick your dog! But there are multiple breeds to choose from. How do you know which breed is best for you? Keep reading to find out what you should consider when picking your first dog.
+
+Lifestyle
+
+What kind of lifestyle do you lead? Do you work multiple jobs? Are you often on the go? 
+
+Some breeds, like the Cavalier King Charles Spaniel, Golden Retrievers, and Border Collie, are companion animals at heart. In other words, they can develop separation anxiety. If you're usually on the go and rarely at home, you may want to avoid these breeds. Labrador Retrivers, French Bulldogs, and Pugs usually do better when left at home alone.
```
Comment (Begster) [inline content/blog/tips-for-your-first-dog.md]: Typo: Retrievers

**Review by Begster — APPROVED** [review]
Very nice write up, Deanna! I very much like your writing style. 
Thanks for allowing me to review. I hope I did it correctly! 🙂

File: content/blog/tips-for-your-first-dog.md (hunk 3 above)
Comment (itsdeannat) [reply content/blog/tips-for-your-first-dog.md:11]: Thank you :)

File: content/blog/tips-for-your-first-dog.md (hunk 4 above)
Comment (itsdeannat) [reply content/blog/tips-for-your-first-dog.md]: Fixed, thank you!

**Review by barbaricyawps — APPROVED** [review]
Approved!

**Pull request merged**